checks.d/elastic.py

```diff
@@ -126,7 +159,7 @@ class ESCheck(AgentCheck):
         "elasticsearch.relocating_shards": ("gauge", "relocating_shards"),
         "elasticsearch.initializing_shards": ("gauge", "initializing_shards"),
         "elasticsearch.unassigned_shards": ("gauge", "unassigned_shards"),
-        "elasticsearch.cluster_status": ("gauge", "status", lambda v: {"red":0, "yellow":1, "green":2}.get(v, -1)),
+        "elasticsearch.cluster_status": ("gauge", "status", lambda v: {"red": 0, "yellow": 1, "green": 2}.get(v, -1)),
```

Comment:
What class constants ? This is a mapping that is not related to service checks. This is send as a metric, it was done before we had service checks so one could alert on the cluster status.

Those values are actually the opposite of the constants we use for service checks.